### DISCUSSION TITLE
Cask

### DISCUSSION THREAD

**Discussion by Priya7242** [Homebrew/discussions#4180] [Casks]
### Output of `brew config`

```shell
Failed to connect to raw.githubusercontent.com port 443 after 75227 ms: Couldn't connect to server
```

### Output of `brew doctor`

```shell
Failed to connect to raw.githubusercontent.com port 443 after 75227 ms: Couldn't connect to server
```

### Description of issue

Issue 

### Relevant casks

Failed to connect to raw.githubusercontent.com port 443 after 75227 ms: Couldn't connect to server

**Comment by Rylan12** [comment]
There is an issue on GitHub's end with how we were distributing formulae. We've temporarily rolled that feature back until GitHub fixes the issue. Run `brew update` and then things should start to work for you again 🤞
